code/modules/blob/blob.dm

```diff
@@ -233,56 +196,107 @@
+			to_chat(user, SPAN_NOTICE("You collect a sample from \the [src]."))
+			var/obj/P = new product(get_turf(user))
+			user.put_in_hands(P)
+			pruned = TRUE
+			return
+		else
+			to_chat(user, SPAN_WARNING("\The [src] has already been pruned."))
+			return
+
 	var/damage = 0
 	switch(W.damtype)
-		if("fire")
+		if(BURN)
 			damage = (W.force / fire_resist)
 			if(W.iswelder())
-				playsound(loc, 'sound/items/Welder.ogg', 100, 1)
-		if("brute")
-			if(prob(30) && !issilicon(user))
-				visible_message("<span class='danger'>\The [W] gets caught in the gelatinous folds of \the [src]</span>")
-				user.drop_from_inventory(W,get_turf(src))
-				return
+				playsound(get_turf(src), 'sound/items/Welder.ogg', 100, TRUE)
+		if(BRUTE)
 			damage = (W.force / brute_resist)
 
 	take_damage(damage)
 	return
 
+#define CORE_SHIELD_HIGH "high"
+#define CORE_SHIELD_LOW "low"
+
 /obj/effect/blob/core
-	name = "blob core"
+	name = "master nucleus"
+	desc = "A massive, fragile nucleus guarded by a shield of thick tendrils."
 	icon_state = "blob_core"
-	light_range = 1
-	light_power = 2
-	light_color = "#F3D203"
-	maxHealth = 200
-	brute_resist = 2
-	laser_resist = 7
-	regen_rate = 2
-	fire_resist = 2
-	var/core_count //amount of secondary cores
-	var/core_limit = 4 //for if a badmin ever wants the station to die, they can set this higher
-	var/blob_count = 1 //amount of actual blob pieces
-	var/blob_limit = 150 //9x4+100 + a bit, maximum amount of blobs allowed.
-
+	maxHealth = 450
+	damage_min = 25
+	damage_max = 35
 	expandType = /obj/effect/blob/shield
-
-
-/obj/effect/blob/core/New()
-	if(!parent_core)
-		parent_core = src
-	..()
-
+	product = /obj/item/blob_tendril/core
+	is_core = TRUE
+
+	light_color = BLOB_COLOR_CORE
+	layer = BLOB_CORE_LAYER
+
+	var/blob_may_process = TRUE
+	var/reported_low_damage = FALSE
+	var/pulse_power = 50 // How far the core can potentially grow in size
+	var/times_to_pulse = 4
+
+/obj/effect/blob/core/proc/get_health_percent()
+	return ((health / maxHealth) * 100)
+
+/*
+the master core becomes more vulnereable to damage as it weakens,
+but it also becomes more aggressive, and channels more of its energy into regenerating rather than spreading
+regen() will cover update_icon() for this proc
+*/
+/obj/effect/blob/core/proc/process_core_health()
+	switch(get_health_percent())
+		if(75 to INFINITY)
+			brute_resist = 3.5
+			fire_resist = 2
+			attack_cooldown = 60
+			regen_rate = 2
+			times_to_pulse = 4
+			if(reported_low_damage)
+				report_shield_status(CORE_SHIELD_HIGH)
+		if(50 to 74)
+			brute_resist = 2.5
+			fire_resist = 1.5
+			attack_cooldown = 50
+			regen_rate = 3
+			times_to_pulse = 3
+		if(34 to 49)
+			brute_resist = 1
+			fire_resist = 0.8
+			attack_cooldown = 40
+			regen_rate = 4
+			times_to_pulse = 2
+		if(-INFINITY to 33)
+			brute_resist = 0.5
+			fire_resist = 0.3
+			attack_cooldown = 30
+			regen_rate = 5
+			times_to_pulse = 1
+			if(!reported_low_damage)
+				report_shield_status(CORE_SHIELD_LOW)
```

Comment:
You can, pretty much, simplify this by just making them calculations based off of blob HP. This monolith switch is really not a good way to do it.